frontends/mit-open/src/page-components/TestimonialCard/TestimonialCard.tsx

```diff
@@ -0,0 +1,184 @@
+import React from "react"
+
+import { ActionButton, styled, theme } from "ol-components"
+import { useTestimonialList } from "api/hooks/testimonials"
+import { Attestation } from "api/v0"
+import { RiArrowRightLine, RiArrowLeftLine } from "@remixicon/react"
+import Slider from "react-slick"
+
+type TestimonialCardProps = {
+  channels?: Array<number>
+  offerors?: Array<string>
+}
+
+type InternalTestimonialCardProps = {
+  resource: Attestation
+}
+
+const QuoteContainer = styled.section(({ theme }) => ({
+  backgroundColor: theme.custom.colors.darkGray2,
+  color: theme.custom.colors.white,
+  overflow: "auto",
+  padding: "16px 0 24px 0",
+}))
+
+const QuoteBlock = styled.div(() => ({
+  justifyContent: "center",
+  alignItems: "center",
+  width: "100%",
+  maxWidth: "1328px",
+  margin: "0 auto",
+  padding: "0 22px",
+}))
+
+const QuoteLeader = styled.div(({ theme }) => ({
+  width: "100%",
+  height: "32px",
+  fontSize: "60px",
+  lineHeight: "60px",
+  color: theme.custom.colors.silverGray,
+}))
+
+const QuoteBody = styled.div(({ theme }) => ({
+  width: "100%",
+  display: "flex",
+  [theme.breakpoints.down("md")]: {
+    flexDirection: "column",
+  },
+}))
+
+const AttestationBlock = styled.div(({ theme }) => ({
+  width: "auto",
+  flexGrow: "5",
+  ...theme.typography.h5,
+  justifyContent: "top",
+}))
+
+const AttestantBlock = styled.div(({ theme }) => ({
+  width: "248px",
+  display: "flex",
+  [theme.breakpoints.down("md")]: {
+    marginTop: "24px",
+  },
+}))
+
+const AttestantAvatar = styled.div({
+  marginRight: "12px",
+  img: {
+    objectFit: "cover",
+    borderRadius: "40px",
+    width: "40px",
+    height: "40px",
+    boxShadow:
+      "0px 2px 4px 0px rgba(37, 38, 43, 0.10), 0px 2px 4px 0px rgba(37, 38, 43, 0.10)",
+  },
+})
+
+const AttestantNameBlock = styled.div(({ theme }) => ({
+  flexGrow: "1",
+  width: "auto",
+  color: theme.custom.colors.lightGray2,
+}))
+
+const AttestantName = styled.div(({ theme }) => ({
+  ...theme.typography.subtitle1,
+  whiteSpace: "nowrap",
+  color: theme.custom.colors.white,
+  lineHeight: "125%",
+}))
+
+const ButtonsContainer = styled.div(({ theme }) => ({
+  display: "flex",
+  justifyContent: "right",
+  margin: "5px auto 0",
+  gap: "16px",
+  [theme.breakpoints.down("md")]: {
+    marginTop: "16px",
+  },
+}))
+
+const InteriorTestimonialCard: React.FC<InternalTestimonialCardProps> = ({
+  resource,
+}) => {
+  return (
+    <QuoteBody>
+      <AttestationBlock>
+        {resource?.quote ??
+          "You have three powerful forces coming together again, the HBCUs, and MIT, Merlot. The need is worldwide for people of all races, underrepresented populations to be able to access information, and to tap in to rich cultural resources to help them in their academic journey."}
```

Comment:
`quote`, `attestant_name`, `title` are all required, right? Let's drop the fallbacks.